addons/fastroping/functions/fnc_deployAI.sqf

```diff
@@ -0,0 +1,95 @@
+/*
+ * Author: BaerMitUmlaut
+ * Auomatically deploy a helicopter filled with AI units.
+ *
+ * Arguments:
+ * 0: The helicopter <OBJECT>
+ * 1: Deploy special roles (gunners, copilot) (default: false) <BOOL>
+ * 2: Create deployment group (default: true) <BOOL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_vehicle] call ace_fastroping_fnc_deployAI
+ *
+ * Public: Yes
+ */
+
+#include "script_component.hpp"
+params [["_vehicle", objNull, [objNull]], ["_deploySpecial", false, [true]], ["_createDeploymentGroup", true, [true]]];
+private ["_config", "_configEnabled", "_deployTime", "_unitsToDeploy", "_deployGroup"];
+
+if (isNull _vehicle || {!(_vehicle isKindOf "Helicopter")}) exitWith {
+    if (hasInterface) then {
+        ["deployAI was called with an invalid or non-existant vehicle.", QFUNC(deployAI)] spawn BIS_fnc_guiMessage;
```

Comment:
Do we really want to show a clickable error message if somebody places a waypoint of this type by mistake?